### ISSUE TITLE
ConfigParser include error

### ISSUE THREAD

**Issue by 7ranceaddic7** [nicolargo/glances#753]
I'm sure I'm doing something wrong some where but I can't figure it out.  Ideas?

scott@Jupiter:~$ glances --export-influxdb
Traceback (most recent call last):
  File "/usr/bin/glances", line 9, in <module>
    load_entry_point('Glances==2.3', 'console_scripts', 'glances')()
  File "/usr/lib/python3/dist-packages/glances/__init__.py", line 119, in main
    args=core.get_args())
  File "/usr/lib/python3/dist-packages/glances/core/glances_standalone.py", line 36, in __init__
    self.stats = GlancesStats(config=config, args=args)
  File "/usr/lib/python3/dist-packages/glances/core/glances_stats.py", line 60, in __init__
    self.load_exports(args=args)
  File "/usr/lib/python3/dist-packages/glances/core/glances_stats.py", line 125, in load_exports
    export_module = __import__(os.path.basename(item)[:-3])
  File "/usr/lib/python3/dist-packages/glances/exports/glances_influxdb.py", line 28, in <module>
    from ConfigParser import NoSectionError, NoOptionError
ImportError: No module named 'ConfigParser'

**Comment by nicolargo** [comment]
Need more information to investigate, please read and follow the guide line to report issue: https://github.com/nicolargo/glances/blob/master/CONTRIBUTING.md

**Comment by asergi** [comment]
The influxdb module wasn't python3-compatible at that time (Glances 2.3).
Please upgrade.

**Comment by subodhp** [comment]
I am facing error - 

Traceback (most recent call last):
  File "/usr/bin/glances", line 9, in <module>
    load_entry_point('Glances==2.3', 'console_scripts', 'glances')()
  File "/usr/lib/python3/dist-packages/glances/__init__.py", line 119, in main
    args=core.get_args())
  File "/usr/lib/python3/dist-packages/glances/core/glances_standalone.py", line 36, in __init__
    self.stats = GlancesStats(config=config, args=args)
  File "/usr/lib/python3/dist-packages/glances/core/glances_stats.py", line 60, in __init__
    self.load_exports(args=args)
  File "/usr/lib/python3/dist-packages/glances/core/glances_stats.py", line 125, in load_exports
    export_module = __import__(os.path.basename(item)[:-3])
  File "/usr/lib/python3/dist-packages/glances/exports/glances_influxdb.py", line 28, in <module>
    from ConfigParser import NoSectionError, NoOptionError
ImportError: No module named 'ConfigParser'

I have the configparser/Configparser installed already on the system. 

I am trying to use Glances + Influxdb + Grafana.

Glances v2.3 with psutil v3.4.2
influxDB version 0.10.0

**Comment by subodhp** [comment]
Since its Python3, I have changed ConfigParser to configparser in file "/usr/lib/python3/dist-packages/glances/exports/glances_influxdb.py", so the ConfigParser error is gone now.
Then I got "AttributeError: 'InfluxDBClient' object has no attribute 'get_database_list'".
I had the Influxdb installed from Aptitude.

Then I followed the sudo pip3 install --upgrade influxdb

still I am getting the same error for get_database_list.

Any pointers?

Thanks,
Subodh Pachghare
thesubodh.com
